packages/fxa-content-server/app/scripts/lib/cropper.js

```diff
@@ -44,19 +39,37 @@ function Cropper(options) {
   if (!options.container) {
     throw new Error('A container element is required');
   }
+
   this._setupElements(options.container);
 
+  window.addEventListener(
+    'resize',
+    _.throttle(() => {
+      this.updateMeasurements();
+
+      const pos = this.updatePosition();
+      this.img.css(pos);
+    }, 300)
+  );
+
   if (options.src) {
-    var img = options;
+    let img = options;
 
     // we want the extra resolution to be able to zoom in later (up to 200%)
-    var maxLength = this.exportLength * 2;
+    const maxLength = this.exportLength * 2;
     if (img.width > maxLength || img.height > maxLength) {
-      var scale = maxLength / Math.max(img.width, img.height);
+      const scale = maxLength / Math.max(img.width, img.height);
       img = this.resize(img, scale);
     }
     this.setImageSrc(img.src, img.width, img.height);
   }
+
+  setTimeout(() => {
```

Comment:
I spent more time than I'd like to admit trying to debug an issue where the cropper is checking for element rect values before the DOM was finished laying out elements, so this is the result -- forcing the cropper to perform its checks after initial layout and execution is finished.